app/src/main/java/com/javahelp/frontend/ClientRegistrationActivity.java

```diff
@@ -1,15 +1,133 @@
 package com.javahelp.frontend;
 
+import androidx.annotation.ColorInt;
 import androidx.appcompat.app.AppCompatActivity;
+
+import android.annotation.SuppressLint;
+import android.content.Intent;
+import android.graphics.Color;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.TextView;
+
 import com.javahelp.R;
 
+import java.util.ArrayList;
+
 
 public class ClientRegistrationActivity extends AppCompatActivity {
 
+    EditText password, confirmpassword, firstname, lastname, user, email, home, phone;
+    TextView passwordAlert, confirmAlert, emailAlert, userAlert, firstAlert, lastAlert, homeAlert, phoneAlert;
+    private Button buttonsignup;
+
+    @SuppressLint("MissingInflatedId")
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_client_registration);
+        //find password, confirm password, name, email id, home address, phone, signup button
+        user = findViewById(R.id.username);
```

Comment:
for this, we can use data binding to make our code cleaner

Reply:
I'm doing this

Reply:
Oh I see. Yeah that would make things cleaner. Let me just make a new commit.